xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-wiki/xwiki-platform-realtime-wiki-ui/pom.xml

```diff
@@ -96,5 +96,11 @@
       <version>${project.version}</version>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>xwiki-platform-search-webjar</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
Where exactly is search suggest used by the Realtime Wiki Editor?

Reply:
I'm not entirely sure why this is here, but reviewing it now it looks like a mistake where I edited the wrong `pom.xml`...
`xwiki-platform-search-ui` is missing the dependency for `search.css`.